script/Ciel/Day 10/10_01_CIEL10_8.txt

```diff
@@ -272,12 +307,12 @@ From the very beginning, Senpai and I lived in different worlds.
 [sha:25f68faea07ad50cce3f6db8ccf96524cfbe47d8]{
 -- Page 607, Offset 28767.
 -- だからあそこで別れるのが最良だと言い聞かせた。
-That's why I said that parting was ideal there.
+That's why I did my best to say goodbye.
 }
 [sha:ef58b2e086d2fc745470693ea9d5b2cf1fe2a212]{
 -- Page 607, Offset 28768.
 -- ―――あの人に嫌われたくなくて、そんな理解を示したつもりでいた。
-I acted as though I understood that, because I didn't want her to hate me.
+I didn't want her to hate me, so I tried to be positive right up until the end.
 }
 [sha:0587c7c77223acea631e84eff65b192aff906937]{
 -- Page 608, Offset 28769.
```

Comment:
Feels like it could be more emphatic

>And where has that led me?

>Just look at me now.

Well, you know, something like that.